### ISSUE TITLE
[FireMonkey] "Get Data from Steam / SteamDB / EpicDB" script doesn't work in FireMonkey

### ISSUE THREAD

**Issue by AsterITA** [erosman/support#352]
Hi, I have been using the script [Get Data from Steam / SteamDB / EpicDB](https://github.com/Sak32009/GetDLCInfoFromSteamDB/) for a long time, recently I noticed that it doesn't work anymore with FireMonkey but works fine with Violent Monkey.

What could be the issue?

**Comment by erosman** [comment]
Does it not work at all or does it work partially?
Are there any errors?

It is a fairly large userscript and not easy to debug for someone not familiar with it.
You can ask the author who knows the userscript.
If you also ask why it doesn't work with Greasemonkey, it might give us a clue.

**Comment by AsterITA** [comment]
> Does it not work at all or does it work partially?

No, it doesn't work at all, I can't see the purple link to click on at the bottom right of the screen, it should be like this: 
![immagine](https://user-images.githubusercontent.com/7688157/130402542-bd50843b-aa9a-416e-8b3f-71463e8d9521.png)

After clicking on that purple link it should redirect on a page like this:
![immagine](https://user-images.githubusercontent.com/7688157/130402780-f7d79a4d-f19d-4d76-a077-94f9edaa6977.png)

If I go to that page it shows me this:
![immagine](https://user-images.githubusercontent.com/7688157/130402814-6b902167-b766-401b-a370-cd3264473217.png)

> Are there any errors?

I don't see any errors on the developer's console.

**Comment by Sak32009** [comment]
Hi, it's a problem I can't identify because absolutely everything works for me and I wonder what it could be every time I see the issue.

**Comment by pintassilgo** [comment]
Tested with FireMonkey v2.34 and it also works for me.
